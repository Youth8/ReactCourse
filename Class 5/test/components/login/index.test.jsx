import "../../__mocks__/epc-ui-layout-mock";
import "../../__mocks__/epc-ui-validation-mock";
import React from "react";
import { render } from "enzyme";
import Login from "../../../src/components/login";

describe("Tests for Login", () => {
  test("Sign in page", () => {
    const instance = render(<Login changeAuth={jest.fn()} />, {
      context: { changeContext: jest.fn() }
    });
    expect(instance).toMatchSnapshot();
  });
});
