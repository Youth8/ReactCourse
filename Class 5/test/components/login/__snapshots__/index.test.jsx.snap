// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Login Sign in page 1`] = `
initialize {
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
